### ISSUE TITLE
error when calling macro referencing a qualified js symbol from a $default namespace

### ISSUE THREAD

**Issue by ikappaki** [babashka/nbb#312]
**version**

v1.2.168

**platform**

win11

**problem**

error when calling a macro referencing a $default namespace symbol:
```Could not resolve symbol: nbb.internal.moment$default/now```

**repro**

1. Create `nbb.edn`
```  clojure
{:paths ["src"]}
```
2. initalize node project and install `moment`
``` bash
npm init

npm i moment
```
3. Create a `src/macros.cljs` to test `$default` namespace calls
``` clojure
(ns macros
  (:require ["moment$default" :as mmt]))

(.now mmt)
;; => 1677315062961

;; (issue.case 1) Can't call instance method on $default namespace
;;
(defmacro mm-1 []
  `(.now mmt))
#_(mm-1)
;; => Could not find instance method: now
(macroexpand-1 '(mm-1))
;; => (.now "nbb.internal.moment$default")

(mmt/now)
;; => 1677315113666

;; (issue.case 2) Neither can call fn on $default namespace
(defmacro mm-2 []
  `(mmt/now))
#_(mm-2)
;; => Could not resolve symbol: nbb.internal.moment$default/now
(macroexpand-1 '(mm-2))
;; => (nbb.internal.moment$default/now)

```
4.  Notice the two cases it fails to call a fn from that namespace.

**expected behavior**

Should be able to use the `$default` namespace in macros as any other js library namespace.

Thanks,

(this is a follow up from #311)

**Comment by borkdude** [comment]
Fixed in 1.2.169

**Comment by ikappaki** [comment]
Thanks a lot! it works like a charm now 🙇
